src/libcollections/linked_list.rs

```diff
@@ -10,8 +10,13 @@
 
 //! A doubly-linked list with owned nodes.
 //!
-//! The `LinkedList` allows pushing and popping elements at either end and is thus
-//! efficiently usable as a double-ended queue.
+//! The `LinkedList` allows pushing and popping elements at either end
+//! in constant time.
+//!
+//! Almost always it is better to use [`Vec`](../vec/struct.Vec.html) or
+//! [`VecDeque`](./vec_deque/struct.VecDeque.html) instead of `LinkedList`.
```

Comment:
Since this is a module-level doc comment, I guess `LinkedList` should have an url.